### ISSUE TITLE
Feature request: Show Storage path in KopiaUI's Repository tab

### ISSUE THREAD

**Issue by okoetter** [kopia/kopia#3418]
I mainly use KopiaUI and got to a point where I forgot where exactly my repository was located.
With CLI a `kopia repository status` shows "Storage config" with its path.
Can we please show this info also in KopiaUI?

**Comment by lupusA** [comment]
Hi, 

you mean the path of the config file?

![image](https://github.com/kopia/kopia/assets/37236531/4d4ea752-604b-44a3-8b7f-2f183b8c3dd2)

Cheers,

**Comment by okoetter** [comment]
No, not the config file but the repository path as `kopia repository status` outputs:

![grafik](https://github.com/kopia/kopia/assets/4048240/a7307921-74cc-4cbc-bf9e-4e073227dfd8)

**Comment by okoetter** [comment]
> The storage path is currently only available, if the repository is a direct repository

Well, that would be sufficient for me ;)

**Comment by cprcrack** [comment]
I have a similar request, maybe they could be combined. When my repository is an external drive and this one is not connected, or if it's an online repository and I have no internet, KopiaUI perpetually shows <initializing> or something similar on the menu. I think it should show the name of the repository so that you know which repository is in what state. This information is locally available on the repository.config file. So instead of `<initializing>` I suggest it shows `My Favorite Repository: <initializing>`.


<img width="357" alt="Screenshot 2023-11-08 at 16 19 52" src="https://github.com/kopia/kopia/assets/650925/f6ac1a00-b441-4cb2-ab71-30f6e4dcda1f">

I would also suggest to show the repository name on KopiaUI window itself, because if you have two Initializing repositories, it's impossible to know which KopiaUI window belongs to which repository. Same change as in the menu, instead of `Initializing Repository...` it should show the name with the information available in the repository.config file: `My Favorite Repository: Initializing Repository...`

<img width="672" alt="Screenshot 2023-11-08 at 16 31 15" src="https://github.com/kopia/kopia/assets/650925/a473da91-57f2-4dfd-b6c9-2de7f5d76349">

**Comment by cprcrack** [comment]
Showing the repo type, path in case of filesystem repos, bucketId or equivalent in case of online repos, would also be great by the way. Currently it feels like KopiaUI doesn't display enough info about the repositories its handling and it can get confusing if you don't open KopiaUI for some time.
